src/specs/api.v2.yaml

```diff
@@ -774,10 +832,29 @@ paths:
               schema:
                 $ref: '#/components/schemas/HTTPError'
   '/experiments/{experimentId}/samples/{sampleId}/metadataTracks/{metadataTrackKey}':
+    parameters:
+      - name: experimentId
+        in: path
+        description: ID of experiment to update.
+        required: true
+        schema:
+          type: string
+      - name: sampleId
+        in: path
+        description: ID of the sample to update.
+        required: true
+        schema:
+          type: string
+      - name: metadataTrackKey
+        in: path
+        description: Key of the metadata track to update.
+        required: true
+        schema:
+          type: string
     patch:
       summary: Updates the value that a sample has in a metadata track in particular
       description: Updates a metadata tracks properties
-      operationId: patchMetadataTrack
+      operationId: patchSampleMetadata
```

Comment:
Nice! by the way, maybe could we make it `patchSampleInMetadataTrackValue`? Or change `metadataTrack#patchSampleInMetadataTrackValue ` to `metadataTrack#patchSampleMetadata`

Reply:
changed